README.md

```diff
@@ -47,7 +47,7 @@ This will launch the interactive prompt. Type `help` to get a list of commands.
 
 ## Notes:
 * If you want to run your own server, please see [zingo lightwalletd](https://github.com/zingolabs/lightwalletd), and then run `./zingo-cli --server http://127.0.0.1:9067`
-* The log file is in `~/.zcash/zingo-wallet.debug.log`. Wallet is stored in `~/.zcash/zingo-wallet.dat`
+* The default log file is in `~/.zcash/zingo-wallet.debug.log`. A default wallet is stored in `~/.zcash/zingo-wallet.dat`
```

Comment:
I don't think we want to play in `~/.zcash`? `~/.zingo`?

Reply:
Out of the scope of this feature.